build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
     implementation 'io.github.cdimascio:dotenv-java:2.2.0'
 
     implementation 'org.apache.httpcomponents:httpclient:4.5.13'
+    implementation 'org.apache.commons:commons-collections4:4.4'
```

Comment:
We should try to avoid adding extra dependencies for such a simple case. I'm sure that we already have the same cases in the existing code